src/Core/Settings.cpp

```diff
@@ -5789,6 +5789,10 @@ Build local plan for local replica
 )", BETA) \
     DECLARE(Bool, parallel_replicas_index_analysis_only_on_coordinator, true, R"(
 Index analysis done only on replica-coordinator and skipped on other replicas. Effective only with enabled parallel_replicas_local_plan
+)", BETA) \
+    \
+    DECLARE(Bool, parallel_replicas_only_with_analyzer, true, R"(
```

Comment:
Please also add setting changes.

Reply:
> Please also add setting changes.

Yeah, thanks. Also, our tests will remind me about it - https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=76427&sha=b60b1d10a0e1caa399fef926d46ed2b072ad7c82&name_0=PR&name_1=Fast+test&name_2=02995_new_settings_history

Reply:
Getting reminded twice is better than none! :smile: but I am happy that we can count on our CI.